src/index.ts

```diff
@@ -19,26 +21,54 @@ const TYPENAME_PAGE = 'page';
  * @param ctx - access to the server context, specifically the schema
  * @returns JS object containing the metadata and content of the retrieved file
  */
-const getSourceFile = async (id, ctx, typename) => {
+const getSourceFile = async (id: string, ctx: any, typename: string): Promise<any> => {
```

Comment:
Would you mind extracting some of these functions out to a `utils/` folder and `resolvers/` folder?